Docs/ProjectSpec.md

```diff
@@ -93,6 +93,7 @@ Note that target names can also be changed by adding a `name` property to a targ
   - `bottom` - at the bottom, after other files
   - `none` - sorted alphabetically with all the other files
 - [ ] **transitivelyLinkDependencies**: **Bool** - If this is `true` then targets will link to the dependencies of their target dependencies. If a target should embed its dependencies, such as application and test bundles, it will embed these transitive dependencies as well. Some complex setups might want to set this to `false` and explicitly specify dependencies at every level. Targets can override this with [Target](#target).transitivelyLinkDependencies. Defaults to `false`.
+- [ ] **generateEmptyDirectories**: **Bool** - If this is `true` then empty directories will be added to project too else will be missed. Defaults to `true`.
```

Comment:
We'll have to keep the default as false for backwards compatibility

Reply:
Yes, you are right, I'll change it to `false`.